deploy-jenkins/templates/jenkins-persistent-template.j2

```diff
@@ -294,6 +294,12 @@
       "displayName": "Jenkins Image Version",
       "description": "Image Version to be used in the Jenkins Container",
       "value": "3.6.0"
+    },
+    {
+      "name": "JENKINS_PASSWORD",
+      "displayName": "User password",
+      "description": "Password for default admin user when not using oauth",
+      "value": "{{ jenkins_pass| default ('password') }}"
```

Comment:
This new parameter - JENKINS_PASSWORD doesn't appear anywhere in the template, how is it used ?

Reply:
@laurafitzgerald wdyt ? ^

Reply:
@matskiv :eyes:

Reply:
yes I think this can be remove. I think this is the same as the change in nagios template just in the wrong place

Reply:
This should have been passed as env var also I missed that so will put both back in